infrastructure/src/main/java/org/corfudb/infrastructure/ServerHandshakeHandler.java

```diff
@@ -62,47 +66,31 @@ public ServerHandshakeHandler(UUID nodeId, String corfuVersion, String timeoutIn
      */
     @Override
     public void channelRead(ChannelHandlerContext ctx, Object m) throws Exception {
+        if (this.handshakeState.failed()) return;
```

Comment:
I noticed that some other 1-line if clause below are inside curly braces, maybe we should add to make style consistent?

Reply:
Done